app/models/challenge.rb

```diff
@@ -94,8 +94,8 @@ def stage_number
       name: 'solution',
       alias: 'story',
       action: 'share',
-      icon: 'fa-play-circle-o',
-      headline: 'Implement and Share Stories',
+      icon: 'fa-paper-plane-o',
```

Comment:
You'll want to find / replace all the references that existed to `fa-play-circle-o` – there are a few more than just this one.